doc/syslog/syslog-design.md

```diff
@@ -195,48 +244,70 @@ The `rsyslog-config` service performs the next actions:
 1. Renders `rsyslog.conf.j2` template with `sonic-cfggen` to generate a new `rsyslogd` config file
 2. Restarts `rsyslog` service which triggers `rsyslogd` to load a new config file
 
-### 2.3.2 SSIP parameters
+### 3.3.2. global syslog configuration
 
-**SSIP will have the next parameter mapping:**
+#### 3.3.2.1. Table 4: Global configuration parameters
+| SONiC                 | Rsyslogd | Config DB Schema                           |
+|:----------------------|:---------|:-------------------------------------------|
+| format                | template | SYSLOG_CONFIG\|GLOBAL\|format              |
+| trap                  | priority | SYSLOG_CONFIG\|GLOBAL\|trap                |
+| welf_firewall_name    | trap     | SYSLOG_CONFIG\|GLOBAL\|welf_firewall_name  |
+
+* trap will set rules trap severity if not specified in the rule
```

Comment:
Please share more details on this statement, preferably with an example.

Reply:
added